helm-charts/metal3-deploy/Chart.yaml

```diff
@@ -27,16 +27,19 @@ dependencies:
   - name: media
     version: 0.1.0
     alias: metal3-media
+    condition: global.enable_metal3_media_server
 
   - name: powerdns
     version: 0.1.0
     alias: metal3-powerdns
+    condition: global.enable_metal3_external_dns_with_pdns
 
   # need to add the bitnami chart repo as well
   - name: external-dns
     version: 6.12.1
     repository: "https://charts.bitnami.com/bitnami"
     alias: metal3-external-dns
+    condition: global.enable_metal3_external_dns_with_pdns
```

Comment:
Right we still need to deploy exernal-dns but without the pdns configuration.

Reply:
While I think that scenario is unlikely because a production environment is expected to have the proper DNS in place, I do agree that we should make it more flexible.